components/collective-navbar/ActionsMenu.js

```diff
@@ -132,11 +174,19 @@ export const getContributeRoute = collective => {
   return { route, params };
 };
 
-const CollectiveNavbarActionsMenu = ({ collective, callsToAction, hiddenActionForNonMobile }) => {
-  const hasRequestGrant =
-    [CollectiveType.FUND].includes(collective.type) || collective.settings?.fundingRequest === true;
+const CollectiveNavbarActionsMenu = ({
+  collective,
+  callsToAction,
+  hiddenActionForNonMobile,
+  mainAction,
+  getActions,
+  isAdmin,
+}) => {
   const enabledCTAs = Object.keys(pickBy(callsToAction, Boolean));
-  const isEmpty = !hasRequestGrant && enabledCTAs.length < 1;
+  const isEmpty = enabledCTAs.length < 1;
+  const hasOnlyTwoCTAs = enabledCTAs.length === 2;
+  const secondAction = without(enabledCTAs, mainAction);
```

Comment:
Variable name doesn't describe what it is, this is not an action but an array of actions. What about:
```suggestion
  const actionsWithtoutMain = without(enabledCTAs, mainAction);
```